tests/test_kcb.py

```diff
@@ -220,14 +220,21 @@ def test_osbs_build(self, tmpdir, pkg_info, failure, orchestrator):
                                                          params='params',
                                                          session=session,
                                                          options=options,
-                                                         workdir='workdir')
+                                                         workdir='workdir',
+                                                         demux=orchestrator)
 
         (flexmock(task)
             .should_receive('fetchDockerfile')
             .with_args(src['src'])
             .and_return(folders_info['dockerfile_path']))
         (flexmock(task)
             .should_receive('_write_incremental_logs'))
+        if orchestrator:
+            (flexmock(task)
+                .should_receive('_write_demultiplexed_logs'))
+        else:
+            (flexmock(task)
+                .should_receive('_write_combined_log'))
```

Comment:
All of these `should_receive()` calls *mask* the methods they name. Instead of calling those methods, flexmock uses a stub. As a result, we are not getting coverage for any of them.

Instead of doing that, I think we want to use flexmock to create a stub in the pretend OSBS client to provide a `get_orchestrator_build_logs` method, and similarly for `get_build_logs`.

Reply:
FYI, flexmock has the concept of spies to track when a method is called but without actually stubbing it out (`should_call` vs `should_receive`) https://has207.github.io/flexmock/user-guide.html#spies-proxies

Reply:
Yeah, I pretty much blindly copied the .should_receive('_write_incremental_logs') bit here, not realizing that these get stubbed out. I'm looking at the `get_orchestrator_build_logs` work done in osbs-client right now to try to get a better idea of what I need to do here.